sdk/storage/azure-storage-blobs/vcpkg/CONTROL

```diff
@@ -0,0 +1,7 @@
+Source: @TARGET_NAME@-cpp
+Version: @AZ_LIBRARY_VERSION@
+Build-Depends: azure-storage-common-cpp
+Description: Microsoft Azure Storage Blobs SDK for C++
+  This library provides Azure Storage Blobs SDK.
+Homepage: https://github.com/Azure/azure-sdk-for-cpp
```

Comment:
Point to blobs path directly instead of the main SDK page

Reply:
Hmm. Does that risk leading to a 404 link if we restructure the repo, or will we always have the blobs path be stable?